fluentd/fluent.conf

```diff
@@ -135,18 +134,36 @@
     bearer_token_file /var/run/secrets/kubernetes.io/serviceaccount/token
     ca_file /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
   </filter>
+  <filter **kibana**>
```

Comment:
Not sure I follow. Why does the message format matter? We're not going to reformat logs for arbitrary applications... can't, really. Just because it says it's kibana doesn't mean it is.